.circleci/cimodel/data/pytorch_build_data.py

```diff
@@ -92,13 +92,6 @@
                 ]),
             ]),
         ]),
-        ("rocm", [
-            ("3.7", [
```

Comment:
Can you update it to 3.8 rather than deleting it, otherwise we would loose a build sanity signal on Circle

Reply:
Updated to 3.9.